.github/workflows/visual-diff.yml

```diff
@@ -0,0 +1,27 @@
+name: visual diff
+on:
+  # This allows PRs opened from forks to execute the Visual Snapshots
+  # since secrets are not passed to workflows when triggered via forks
+  # Using workflow_run is less preferred since it executes even when jobs
+  # do not pass
```

Comment:
Looks like we can check that the workflow completed successfully:

https://docs.github.com/en/enterprise-server@3.2/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow

Although if you did this, we would need it at the step level and we would need a step for failures where we make sure the job fails. Ideally we would pass the workflow conclusion to the API so that it can tell the user *why* it failed.